ucte/ucte-converter/src/main/java/com/powsybl/ucte/converter/UcteImporter.java

```diff
@@ -867,76 +864,84 @@ private void mergeXnodeDanglingLines(UcteNetwork ucteNetwork, Network network) {
 
         Set<DanglingLine> danglingLinesToProcess = Sets.newHashSet(network.getDanglingLines());
         while (!danglingLinesToProcess.isEmpty()) {
-            DanglingLine dl1 = danglingLinesToProcess.iterator().next();
-            DanglingLine dl2 = getMatchingDanglingLine(dl1, danglingLinesByXnodeCode);
+            DanglingLine dlToProcess = danglingLinesToProcess.iterator().next();
+            DanglingLine dlMatchingDlToProcess = getMatchingDanglingLine(dlToProcess, danglingLinesByXnodeCode);
```

Comment:
It's a detail, but the name of this variable should be `matchingDlToProcess`

Reply:
Well, you can also consider that it is the dangling line that matches the dangling line to process.
Which one do you prefer ?